packages/modules/src/modules/banners/common/BannerWrapper.tsx

```diff
@@ -172,6 +172,7 @@ const withBannerData = (
                 email,
                 tickerSettings,
                 isSupporter,
+                numArticles,
```

Comment:
This needed to be passed through to so the investigations banner can render the special opt out subheading